data-serving/data-service/schemas/cases.schema.json

```diff
@@ -197,11 +187,7 @@
             }
           },
           "status": {
-            "enum": [
-              "Asymptomatic",
-              "Presymptomatic",
-              "Symptomatic"
-            ]
+            "bsonType": "string"
```

Comment:
I don't think this one will have an Other field

Reply:
Per discussion documented [here](https://docs.google.com/document/d/1u0FUPZ6vG00irfPaw7CLwekWxpeE7n70WlRRnMQAi-g/edit#heading=h.rijffgfpldij), we're going to remove all enums from the schema in order to separate the schema from the data; data values will be validated at the API layer instead